### ISSUE TITLE
Failing to download ISO leaves the empty file in artifacts directory

### ISSUE THREAD

**Issue by oddstr13** [parallelvirtualcluster/pvc-installer#1]
```
$ ./buildiso.sh 
Creating temporary directories... done.
Downloading Debian Live ISO... Error downloading source ISO.
Cleaning up... done.

$ ./buildiso.sh 
Creating temporary directories... done.
Extracting Debian Live ISO files... Error mounting Live ISO file.
Cleaning up... done.
```

https://github.com/parallelvirtualcluster/pvc-installer/blob/836b50c38b837c1534b838459e7ab3242cc55248/buildiso.sh#L89

**Comment by joshuaboniface** [comment]
Good find!

**Comment by joshuaboniface** [comment]
Fixed in 86ce4403c4a9e212b54a4dc02b710ed3a0078679
